torch_xla/csrc/helpers.h

```diff
@@ -16,6 +16,7 @@
 #include "tensorflow/compiler/xla/xla_client/debug_macros.h"
 #include "tensorflow/compiler/xla/xla_client/util.h"
```

Comment:
do we still need this util include for these files?

Reply:
For now, yes:

`xla_client/debug_macros.h` is required for:
- XLA_CHECK

`xla_client/util.h` is required for:
- xla::util::Range